### ISSUE TITLE
TPM PCR0 differs from reconstruction, Fedora 32, XPS 13 9300

### ISSUE THREAD

**Issue by fbouynot** [fwupd/fwupd#2161]
**Describe the bug:**
I'm using a Dell XPS 13 9300 on Fedora 32.
`fwupdmgr get-devices` gives me "TPM PCR0 differs from reconstruction" and a link to fill a bug here.

**fwupd version information**

    # fwupdmgr --version
    client version:	1.4.2
    compile-time dependency versions
    	gusb:	0.3.4
    	efivar:	37
    daemon version:	1.4.2

Please ask me if you need more informations.

[fwupdmgrgetdevices.txt](https://github.com/fwupd/fwupd/files/4746813/fwupdmgrgetdevices.txt)
[fwupdtpmevlog.txt](https://github.com/fwupd/fwupd/files/4746814/fwupdtpmevlog.txt)

**Comment by superm1** [comment]
Yes, duplicate of https://github.com/fwupd/firmware-dell/issues/20
